Community/Tdarr_Plugin_vdka_Tiered_CPU_CRF_Based_Configurable.js

```diff
@@ -189,6 +381,14 @@ const plugin = (file, librarySettings, inputs, otherArguments) => {
   const fullhdCRF = inputs.fullhdCRF ? inputs.fullhdCRF : 24;
   const uhdCRF = inputs.uhdCRF ? inputs.uhdCRF : 28;
   const bframe = inputs.bframe ? inputs.bframe : 8;
+  let use10Bit = '';
+
+  if (!inputs.use10Bit) {
+    response.infoLog += '☑Using 8Bit for colorDepth';
+  } else {
+    use10Bit = '-pix_fmt yuv420p10le';
+    response.infoLog += '☑Using 10Bit for colorDepth \n';
+  }
```

Comment:
Not adding the pix_fmt will just keep the original bit depth. And `use` sounds like it should be a boolean, so should be renamed.